File: JS Advanced/15. Modules, Babel, CommonJS Exercises/04. People and Posts/person.js
class Person{
    constructor(name, age){
        this.name = name;
        this.age = age;
    }

    addToSelector(selector) {
        $(selector)
            .append($('<div>').addClass(`person ${this.name}`)
                .append($('<p>').addClass('name').text(this.name))
                .append($('<p>').addClass('age').text(this.age))
                .append($('<div>').addClass(`posts ${this.name}`))
            );
    }
}

module.exports = Person;

File: C# OOP Basics/01. Defining Classes Exercises/01. Define a Class Person/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

public class Person
{
    private string name;
    private int age;

    public string Name
    {
        get
        {
            return this.name;
        }
        set
        {
            this.name = value;
        }
    }

    public int Age
    {
        get
        {
            return this.age;
        }

        set
        {
            this.age = value;
        }
    }
}
